Makefile

```diff
@@ -88,12 +88,12 @@ min_debug:				#: Minimal build with debugging symbols
 	$(MAKE) build BUILD_DIR=debug
 
 benchmarks-build:
-	$(MAKE) release EXTRA_CMAKE_FLAGS="-DVELOX_BUILD_MINIMAL=ON -DVELOX_BUILD_BENCHMARKS=ON"
+	$(MAKE) release EXTRA_CMAKE_FLAGS=" -DVELOX_ENABLE_PARQUET=ON -DVELOX_BUILD_BENCHMARKS=ON"
```

Comment:
I wonder if it makes sense to keep a lean benchmark and add a new one for TPC-H. E.g. we could have a quick to build and run benchmark for core building blocks available in Velox and a separate longer to build and run benchmark that runs complex queries like TPC-H and TPC-DS.

Reply:
I agree.  Will separate this out.